index.html

```diff
@@ -165,28 +165,15 @@ <h2 id="bp_and-reco">Best Practices, Recommendations, and Gaps</h2>
 
 
 <section>
-<h3 id="resource_wide_defaults">Resource-wide &amp; string-specific defaults</h3>
-<p>Many resources use only a single language and have a consistent base text direction. For efficiency, the following are best practices:</p>
-
-<div class="req" id="bp_default_setting">
-<p class="advisement">Define a rule or a field to provide the default language and base direction for all strings in a given resource.</p>
-</div>
-
-
-<div class="req" id="bp_not_only_default">
-<p class="advisement">Specifications MUST NOT assume that a resource-wide default is sufficient.  Even if a resource-wide setting is available, it must be possible to use string-specific metadata  to override that default.</p>
-</div>
-
-
-<p>Document level defaults, when combined with per-field metadata, can reduce the overall complexity of a given document instance, since the language and direction values don't have to be repeated across many fields. However, they do not solve all language or directionality problems, and so it must be possible to override the default on a string-by-string basis, where necessary.</p>
-
-<p>First-strong heuristics are ineffective when a default direction has been set for all strings, since metadata overrides (intentionally) the value of the first-strong character. Therefore it is necessary to use explicitly provided field data to override the default. Even if an RLM character has been prepended to a string, the default metadata overrides it.</p>
-
+<h3 id="resource_wide_defaults">Resource-wide defaults and string-specific metadata</h3>
 
 <div class="req" id="bp_lang_field_based_metadata">
 <p class="advisement">Use field-based metadata or string datatypes to indicate the language and the base direction for individual <a>localizable text</a> values.</p>
 </div>
 
+
+<p>Use of heuristics to determine language and direction will always fail for certain  cases, and there needs to be a way to override the outcome for those cases. Assignment of <a href="#metadata">metadata</a> (either as a resource-wide default, or in a string-specific label) is an intentional act that overrides the need to  guess the outcome by applying heuristics.</p>
```

Comment:
I would change `language and direction` to `language or direction`. 

I'm not keen on the words `outcome` or `override`. Perhaps:

> Use of heuristics to determine the language or base direction will always fail for certain cases, and there needs to be a way to provide the correct value for those cases. Assignment of metadata (either as a resource-wide default or for a specific string) is an intentional act that removes the need to guess the correct value through the application of heuristics.

Reply:
Changes made.